fe/fe-core/src/main/java/com/starrocks/sql/optimizer/MvRewritePreprocessor.java

```diff
@@ -83,12 +85,22 @@
 import java.util.PriorityQueue;
 import java.util.Queue;
 import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
 import java.util.stream.Collectors;
 
+import static com.google.common.util.concurrent.MoreExecutors.newDirectExecutorService;
 import static com.starrocks.sql.optimizer.OptimizerTraceUtil.logMVPrepare;
 
 public class MvRewritePreprocessor {
     private static final Logger LOG = LogManager.getLogger(MvRewritePreprocessor.class);
+
+    private static final Executor MV_PREPARE_EXECUTOR = Executors.newFixedThreadPool(
+            Runtime.getRuntime().availableProcessors(),
```

Comment:
This is a bit heavy for mv prepare processor tasks, can we reduce it for a less value?

Reply:
How about ` Math.max(4, Runtime.getRuntime().availableProcessors() / 4)`

Reply:
done